superdesk/publish/formatters/newsml_1_2_formatter.py

```diff
@@ -0,0 +1,401 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+
+import time
+import logging
+import xml.etree.ElementTree as etree
+from xml.etree.ElementTree import SubElement
+from eve.utils import config
+from superdesk.publish.formatters import Formatter
+import superdesk
+from superdesk.errors import FormatterError
+from superdesk.metadata.item import ITEM_TYPE, CONTENT_TYPE, EMBARGO, ITEM_STATE, CONTENT_STATE
+from superdesk.metadata.packages import PACKAGE_TYPE, GROUP_ID, REFS, RESIDREF, ROLE, ROOT_GROUP
+from superdesk.utc import utcnow
+from flask import current_app as app
+
+
+logger = logging.getLogger(__name__)
+
+
+class NewsML12Formatter(Formatter):
+    """
+    NewsML 1.2 Formatter
+    """
+    XML_ROOT = '<?xml version="1.0"?><!DOCTYPE NewsML SYSTEM "http://www.aap.com.au/xml-res/NewsML_1.2.dtd">'
```

Comment:
Looking at the core formatters the idea is to have IPTC approved formatters. If thats the case then the catalog should point to IPTC instead of AAP.